scripts/build/Platform/Windows/destroy_cdk_applications.cmd

```diff
@@ -0,0 +1,97 @@
+@ECHO OFF
+REM
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM
+REM
+
+REM Destroy the CDK applcations for AWS gems (Windows only)
+REM Prerequisites:
+REM 1) Node.js is installed
+REM 2) Node.js version >= 10.13.0, except for versions 13.0.0 - 13.6.0. A version in active long-term support is recommended.
+SETLOCAL EnableDelayedExpansion
+
+SET SOURCE_DIRECTORY=%CD%
+SET PATH=%SOURCE_DIRECTORY%\python;%PATH%
+SET GEM_DIRECTORY=%SOURCE_DIRECTORY%\Gems
+
+REM Create and activate a virtualenv for the CDK destruction
+CALL python -m venv .env
+IF ERRORLEVEL 1 (
+    ECHO [cdk_bootstrap] Failed to create a virtualenv for the CDK destruction
+    exit /b 1
+)
+CALL .env\Scripts\activate.bat
+IF ERRORLEVEL 1 (
+    ECHO [cdk_bootstrap] Failed to activate the virtualenv for the CDK destruction
+    exit /b 1
+)
+
+ECHO [cdk_installation] Install the latest version of CDK

```

Comment:
nit: you are uninstalling here.

Reply:
No, I'm actually uninstall the current version and install the latest version of CDK here. There's an known issue where upgrading the latest version directly can cause some schema issues later.

Reply:
why not install latest cdk version at the beginning of the pipe, then we don't need to do it in clean up step? It should be rare that cdk gets updated during automated testing?

Reply:
We do install the latest cdk version at the beginning of the pipe (in the deploy_cdk_application.cmd). Reinstall it here so that users can run the destroy job independently to clean up the resources or verify the destruction behavior.